src/Magnum/ImGuiIntegration/Context.cpp

```diff
@@ -271,6 +325,51 @@ void Context::drawFrame() {
     CORRADE_INTERNAL_ASSERT(drawData); /* This is always valid after Render() */
     drawData->ScaleClipRects(io.DisplayFramebufferScale);
 
+    #if IMGUI_VERSION_NUM >= 19200
+    if(drawData->Textures) {
+        for(ImTextureData* tex : *drawData->Textures) {
+            switch(tex->Status) {
+                case ImTextureStatus_WantCreate: {
+                    /* There's a single-channel format, but most official
+                       backends don't support it either */
+                    CORRADE_INTERNAL_ASSERT(tex->Format == ImTextureFormat_RGBA32);
+                    CORRADE_INTERNAL_ASSERT(tex->GetTexID() == ImTextureID_Invalid);
+                    const Vector2i size{tex->Width, tex->Height};
+                    GL::Texture2D glTexture;
+                    glTexture
+                        .setMinificationFilter(SamplerFilter::Linear)
+                        .setMagnificationFilter(SamplerFilter::Linear)
+                        .setWrapping(GL::SamplerWrapping::ClampToEdge)
+                        .setStorage(1, GL::TextureFormat::RGBA8, size);
+                    const ImTextureID id = ImTextureID(glTexture.release());
+                    tex->SetTexID(id);
+                    updateTexture(*tex, Range2Di::fromSize({}, size));
+                    tex->SetStatus(ImTextureStatus_OK);
+                    break;
+                }
+                case ImTextureStatus_WantDestroy:
+                    destroyTexture(*tex);
+                    /* destroyTexture() sets status and invalid ID */
+                    break;
+                case ImTextureStatus_WantUpdates:
+                    /* There's also the combined rect of all updates in
+                       tex->UpdateRect but the official GL backend doesn't use
+                       it either */
```

Comment:
:eyes: 

I'd honestly prefer the combined rect here, especially given the overhead on WebGL (where it could be doing CPU-side copies and whatnot for each such call) in my opinion makes more sense to optimize for less GL calls than less memory copied.

With Vulkan there's IIRC a way to perform a copy of multiple rectangles in a single call so there it'd be preferred, but with GL I think it's easiest to just fire off one copy and hope for it to be DMA'd in some way.